SOURCES.md

```diff
@@ -136,6 +138,7 @@ http://ftp1.scientificlinux.org/linux/scientific/7x/x86_64/os/RPM-GPG-KEY-sl7
 
 https://www.suse.com/support/security/keys/
 
+This repository contains the ALP (Adaptable Linux Platform) signing keys used also in openSUSE Leap 16.
```

Comment:
ALP shouldn't be present in any public facing infrastructure anymore, so is this note still needed?

Reply:
Yes well, this is the side effect of them not wanting to resign packages when exporting them for openSUSE Leap. 😦